src/components/index.js

```diff
@@ -0,0 +1,3 @@
+/*
+* Insert in this folder your react component.
```

Comment:
I think we could delete this file

Reply:
We can leave the file or use a `.gitkeep` instead; i've no strong opinion about this... Maybe the `.gitkeep` has less impacts on how the user will handle its files.

Reply:
note: this applied to all empty index files in the sibling directories.